shogun-config/src/main/resources/application-base.yml

```diff
@@ -37,6 +37,7 @@ spring:
     baselineOnMigrate: true
     locations: classpath:db/migration
     defaultSchema: shogun
+    outOfOrder: true
```

Comment:
Are you sure we want this? AFAICT this may e.g. apply some migration altering a table before it's actually created by another one

Reply:
Reset in 0b32973049a1a58045c1043b1de758ad06af3597.

Please also note #263.